flow.py

```diff
@@ -89,25 +89,66 @@ def __init__(self, id_=None, start=None, amount=None, src_obj=None, dst_obj=None
 
         self.time_of_last_window_reduction = float('-inf')
 
+        # Smallest sequence number that receiver hasn't received
+        self.next_expected = 0
+        self.packets_received_out_of_order = []
+
+        # Start of transmit window (earliest unacknowledged sequence number)
+        self.transmit_window_start = 0
+
+        # If we eventually do selective acknowledgements
+        #self.packets_ackd_out_of_order = []
+
+        # TCP States (affect how window is changed)
+        self.slow_start = True
```

Comment:
Style: Could combine self.slow_start (boolean) and self.congestion_avoidance (boolean) into self.tcp_state (string)